### ISSUE TITLE
Project page: when entering ctrl+s save project not page

### ISSUE THREAD

**Issue by khoroshevskyi** [pepkit/pephub#220]
can we add functionality to project page, so when you entering ctrl+s it will save project, instead of saving page?

**Comment by nleroy917** [comment]
Yes, that will be easy. Inside the `Project.tsx` page component, we need only add a small event listener to trigger the project update. It will look like this:

```ts
window.addEventListener('keydown', (e) => {
      // detect ctrl + k key press
      if (e.ctrlKey && e.key === 's') {
        e.preventDefault();
        // something like this
        updateProject();
      }
    });
```

It will need to be dependent on the platform (macOS should be `CMD+S` and Windows/Linux should be `ctrl+S`). I'e already done this however for the search hot key, so we can just copy that.

**Comment by nleroy917** [comment]
This is still a good idea and should be added to the milestone for PEPhub week

**Comment by nleroy917** [comment]
This has been added

**Comment by khoroshevskyi** [comment]
This issue was solved in v0.11.0
